### ISSUE TITLE
CycleGAN inference

### ISSUE THREAD

**Issue by Anastasiyabordak** [junyanz/pytorch-CycleGAN-and-pix2pix#922]
Can I get multiple variants from one trained CycleGAN in inference?
For instance:
I have one picture of a horse and I would like to have 4 different(!!!) pictures in style, trained in CycleGAN. 
Is it possible?

**Comment by junyanz** [comment]
You can add dropout to the generator by removing the flag `--no_dropout`. In the test time, it can produce some variations. To get more diverse data, you can have a look at some recent work such as [MUNIT](https://github.com/NVlabs/MUNIT), [DRIT](https://github.com/HsinYingLee/DRIT), and [Augmented CycleGAN](https://github.com/aalmah/augmented_cyclegan).
